docs/modelserving/v1beta1/sklearn/v2/README.md

```diff
@@ -127,7 +127,7 @@ Now, you can use `curl` to send the inference request as:
 ```bash
 SERVICE_HOSTNAME=$(kubectl get inferenceservice sklearn-v2-iris -o jsonpath='{.status.url}' | cut -d "/" -f 3)
 
-curl -v \
+curl -XPOST -v \
```

Comment:
I believe we don't have to specify POST if we are using it with `-d` option

Reply:
ok, you are right, -XPOST can reduce confuse.